### DISCUSSION TITLE
Are there Gruntwork modules that handle DR scenarios by default?

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#523] [Help] [answered]
Do modules for things like S3, RDS, EC2, etc., have disaster recovery built in, or is that primarily the responsibility of the customer? Does Gruntwork have an opinionated way to think about DR?

(terraform-aws-data-storage)
---

<ins datetime="2022-07-28T17:39:24Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109074">Tracked in ticket #109074</a></p>
</ins>

**Comment by zackproser** [comment, marked as answer]
Generally speaking the modules in our terraform-aws-data-storage repository allow you to configure: 

* Deletion protection
* Performance insights / metrics enhancements
* Encryption
* Monitoring
* Backups and snapshots behavior
* And more...

However, our modules don't currently handle Disaster Recovery scenarios by default. While we may consider such enhancements / extensions in the future, we currently feel our focus is better spent ensuring our core modules are up to date and that they cover the most common use cases.
